recaptcha/recaptcha-enterprise/build.gradle.kts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2025 Ping Identity Corporation. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+description = "Recaptcha Enterprise Library"
+
+plugins {
+    id("com.pingidentity.convention.android.library")
+    id("com.pingidentity.convention.jacoco")
+    alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.kotlinAndroid)
+}
+
+android {
+    namespace = "com.pingidentity.recaptcha.enterprise"
+}
+
+dependencies {
+    implementation(project(":foundation:journey-plugin"))
+    implementation(project(":foundation:utils"))
+    implementation(project(":foundation:android"))
+    implementation(project(":foundation:logger"))
+    implementation(libs.androidx.annotation)
+    implementation(libs.androidx.appcompat)
+    implementation(libs.androidx.startup.runtime)
+
+    // Make it optional for developer
+    compileOnly(libs.google.android.recaptcha)
```

Comment:
Make it implementation dependency, when developer make a dependency on this recaptcha module, they want to do recaptcha with google library, currently there is no other option, so I think we can just include the dependency for the application.